### ISSUE TITLE
Error installing

### ISSUE THREAD

**Issue by tshrjn** [amosjyng/langchain-visualizer#1]
`pip install langchain-visualizer` leads no distribution match. Im currently using python 3.11.0

```

ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python >=3.10,<3.11; 0.0.10 Requires-Python >=3.10,<3.11; 0.0.2 Requires-Python >=3.10,<3.11; 0.0.3 Requires-Python >=3.10,<3.11; 0.0.4 Requires-Python >=3.10,<3.11; 0.0.5 Requires-Python >=3.10,<3.11; 0.0.6 Requires-Python >=3.10,<3.11; 0.0.7 Requires-Python >=3.10,<3.11; 0.0.8 Requires-Python >=3.10,<3.11; 0.0.9 Requires-Python >=3.10,<3.11
ERROR: Could not find a version that satisfies the requirement langchain-visualizer (from versions: none)
ERROR: No matching distribution found for langchain-visualizer
```

**Comment by amosjyng** [comment]
Sorry about that, it's because the package `ought-ice`, which this depends on for visualization, only works with Python 3.10 for its current version. However, they've recently implemented support for Python 3.11 as well. I've created [a new issue](https://github.com/oughtinc/ice/issues/269) asking if they're going to make a​ new release for that soon.

In the meantime, would it be possible for you to try this out with Python 3.10 instead? I've found [asdf](https://github.com/asdf-vm/asdf) to be an easy way to switch between versions :)
